### ISSUE TITLE
HTML being shown when mockdate set.

### ISSUE THREAD

**Issue by umagon** [netsells/storybook-mockdate#1]
Type: **Bug**
I am using Storybook 6.5.10 for React

When I use the decorator this way in a `.stories.js` file:

```javascript
import withMockdate from '@netsells/storybook-mockdate';

export default {
  title: "Timeframes Grouping",
  component: TimeframesGrouping,
  parameters: {
    mockdate: new Date(2016, 10, 11)
  },
  decorators: [ withMockdate ]
}
```

All I see is:
<img width="485" alt="image" src="https://user-images.githubusercontent.com/4467191/187288319-2c6879f8-4ba2-4192-b5f9-6faba02a9309.png">

**Comment by samturrell** [comment]
Hi @umagon - this package is only compatible with vue at the moment, but i will look to add react support when i get a moment

**Comment by bryantee** [comment]
I opened up a PR that would address. Uncertain if you think this is a good approach @samturrell. I have not specifically tested with something like `yarn link`, but I am using a very similar implementation in a React code base with success.

**Comment by samturrell** [comment]
Hi @umagon + @bryantee - apologies for the delay on this. 

I have just published `0.2.0` - please check it out when you can and report any issues. Can be added to your projects the same way regardless of framework, and it should auto-detect the framework to use.
